content/blog/introduction-to-typescript/index.md

```diff
@@ -0,0 +1,212 @@
+    age: number;
+}
+```
+
+And this feature doesn't have an API dissimilar to standards-based APIs; [It's being built with and on top of features that are being proposed for a future version of JavaScript (commonly referred to as ESNext).](https://www.typescriptlang.org/docs/handbook/decorators.html#metadata)
+
+# What isn't TypeScript {#misconceptions}
+
+Now that we've covered a bit of what TypeScript _is_, it might be a good idea to quickly synopsis what it _isn't_. After all, to know what something is not is oftentimes just as powerful as knowing what something _is_.
+
+## It's Not the Tower You Think It Is {#typescript-is-not-babel}
+
+One of the things TypeScript is not is a transpiler. What this means is that TypeScript (alone) _will not take TypeScript source code that contains syntax from newer JavaScript versions (ES6+) and output older versions of JavaScript (ES5) in order to improve browser compatibility (IE11)_.
+
+For anyone who's used TypeScript this may confuse you, as there are various flags and config options for the output version of JavaScript it compiles to. What's really happening under-the-hood is that TypeScript hands off your source code to Babel after it compiles down to JavaScript. [With Babel 7 this is even harder to notice](https://devblogs.microsoft.com/typescript/typescript-and-babel-7/) but be aware that any transpilation you expect to occur when using TypeScript may force you to use and understand Babel tools.
+
+However, this does mean that you can utilize the entire arsenal of Babel tooling to your disposal, [such as Babel plugins](https://babeljs.io/docs/en/plugins/).
+
+## Logic != Typings {#typings-are-not-logic}
+
+_TypeScript will not find all your typing errors on its own_. This is because TypeScript is only as useful as your typings are. [Let's look back at an earlier example](#type-safety):
+
+```javascript
+function addFive(input) {
+    return input + 5;
+}
+```
+
+If you keep `addFive`'s `input` parameter without an explicit type, it will try to do its best to detect the type based on the operations you run on the value. **While these inferred types are often very useful, it often has difficulty doing so in an accurate manor**. _This is why manually assigning types is almost always preferred to leaving them be inferred_. If it couldn't properly detect a type for the value in this example, it does nothing to prevent strings from being passed as the parameter value.
+
+Although examples like this are simple, strict typings can also become fairly complex ([see the Advanced section of the official handbook for such examples](https://www.typescriptlang.org/docs/handbook/advanced-types.html)) in order to maintain maximum type strictness.
+Also, because typings do nothing to test against the logic of your program, they should not be seen as a replacement for testing, but rather a companion to them. With strict typings and proper testing, regressions can be severely limited and improve code quality of life.
```

Comment:
Should this be promoted with styling or position within the heading? This was very important for me when first starting out with TS.

Reply:
```suggestion
**Also, because typings do nothing to test against the logic of your program, they should not be seen as a replacement for testing, but rather a companion to them.** _With strict typings and proper testing, regressions can be severely limited and improve code quality of life._
```